source/retryable-reads/tests/changeStreams-db.coll.watch-serverErrors.yml

```diff
@@ -147,3 +147,30 @@ tests:
         operations: [*retryable_operation_fails]
         expectations:
              - *retryable_command_started_event
+    -
+        description: "db.coll.watch succeeds after ExceededTimeLimit"
+        failPoint:
+            <<: *failCommand_failPoint
+            data: { failCommands: [aggregate], errorCode: 262 }
+        operations: [*retryable_operation]
```

Comment:
I missed that these were also inconsistent with `db.watch-serverErrors.yml`. Good catch.